enterprise/api/v1/views.py

```diff
@@ -1483,3 +1484,54 @@ def update(self, request, *args, **kwargs):
     )
     def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
+
+
+class EnterpriseUserLinkView(EnterpriseReadWriteModelViewSet):
+    """
+    View for
+    /enterprise/api/enterprise_user_link/{enterprise_customer_key}
+
+    Given a enterprise_customer_key, link user to the appropriate enterprise.
+
+    If the key is not found, returns 400
+    If the ket is not valid, return 422
+    """
+    permission_classes = (permissions.IsAuthenticated,)
+
+    @action(methods=['post'], detail=False)
+    def link_user(self, request, enterprise_customer_key):
```

Comment:
Curious to hear your thoughts on whether this should be its own view class? An alternative might be for it to be an action on the existing `EnterpriseCustomerInviteKeyViewSet` (since it already has to do with the EnterpriseCustomerInviteKey), where the endpoint URL might be:

```/enterprise/api/v1/enterprise-customer-invite-key/<enterprise_customer_invite_key_uuid>/link-user/```

Reply:
You convinced me! Fixed!